src/components/Feilresponser/revurderingFeilresponser/RevurderingFeilresponser.ts

```diff
@@ -0,0 +1,176 @@
+import { ErrorMessage } from '~api/apiClient';
+import { MessageFormatter } from '~lib/i18n';
+import { Nullable } from '~lib/types';
+
+import messages from './RevurderingFeilresponser-nb';
+
+export const revurderingFeilkodeTilFeilmelding = (
+    formatMessage: MessageFormatter<typeof messages>,
+    feil?: Nullable<ErrorMessage>
+) => {
+    const messageId = revurderingErrorCodeMessageIdMap[(feil?.code ?? '') as RevurderingErrorCodes];
+    return formatMessage(messageId ?? 'feil.ukjentFeil');
+};
+
+export type RevurderingErrorCodes =
+    | GenerellErrors
+    | VurderingsperiodeErrors
+    | VilkårErrorCodes
+    | BeregningErrors
+    | ForhåndsvarslingErrors
+    | UtfallSomIkkeStøttesErrors
+    | BrevErrors;
+
+type VilkårErrorCodes = OpprettelseOgOppdateringErrors | UføreErrors | BostiuasjonErrors | FormueErrors | FradragErrors;
+
+enum GenerellErrors {
+    G_REGULERING_KAN_IKKE_FØRE_TIL_OPPHØR = 'g_regulering_kan_ikke_føre_til_opphør',
+    ATTESTANT_OG_SAKSBEHANDLER_KAN_IKKE_VÆRE_SAMME_PERSON = 'attestant_og_saksbehandler_kan_ikke_være_samme_person',
+    FEILUTBETALING_STØTTES_IKKE = 'feilutbetalinger_støttes_ikke',
+
+    FANT_IKKE_SAK = 'fant_ikke_sak',
```

Comment:
Denne burde vel også gjelde for søknadsbehandling?

Reply:
Per nå er denne bare brukt for revurdering i backend